students/HABTAMU/lesson01/assignment/tests/test_unit.py

```diff
@@ -0,0 +1,105 @@
+""" Unittest Module  """
+
+
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+
+# from inventory_management.electricappliancesclass import ElectricAppliances as elapp
+import inventory_management.electricappliancesclass as e_app_mod
+import inventory_management.furnitureclass as fur_mod
+import inventory_management.inventoryclass as inv_mod
+import inventory_management.management as mgm_mod
+from inventory_management.market_prices import market_prices as mp
+
+
+class Testmanagementclass(TestCase):
+    def test_get_price(self):
+        item_price = mgm_mod.get_price()
+        pass
+
+
+    def test_item_info(self):
+        pass
```

Comment:
As written, item_info is hard to test. Imagine testing this instead:
```
def item_info(item):
    if itemCode in fullInventory:
        printDict = fullInventory[itemCode]
        for k,v in printDict.items():
            return (k, v)
    return None
```
How could you fit this into the application?

Note: this code assumes the user only cares about one of any particular item in inventory, which may not be true.